file-formats/aobj/type/zif_aff_aobj_v1.intf.abap

```diff
@@ -0,0 +1,69 @@
+INTERFACE zif_aff_aobj_v1
+  PUBLIC.
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Write Class</p>
+      "! Name of executable class or program which writes the data to the archive
+      "! $required
+      write_class  TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Delete Class</p>
+      "! Name of executable class or program which deletes the data from the tables
+      "! $required
+      delete_class TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Reload Class</p>
+      "! Name of executable class or program which reloads the data to the tables (may be empty)
+      reload_class TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_attributes.
+
+  TYPES:
+    "! <p class="shorttext">Table to Be Archived</p>
+    "! Tables of business object instance from which data is archived
+    BEGIN OF ty_table_to_be_archived,
+      "! <p class="shorttext"></p>
+      "! Name of table or structure
+      "! $required
+      name TYPE c LENGTH 30,
+    END OF ty_table_to_be_archived.
+
+  "! <p class="shorttext">Tables to Be Archived</p>
+  "! Tables from which data is archived
+  TYPES ty_tables_to_be_archived TYPE SORTED TABLE OF ty_table_to_be_archived WITH UNIQUE KEY name.
+
+  TYPES:
+    "! <p class="shorttext">Name of Archiving Class</p>
+    "! Archiving classes are no ABAP classes, but IDs of reuse objects, whose data should be archived
+    "! Together with the business object instance (for example change documents).
+    BEGIN OF ty_archiving_class,
+      "! <p class="shorttext">Name</p>
+      "! Name of archiving class
+      "! $required
+      name TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_archiving_class.
+  TYPES: ty_archiving_classes TYPE SORTED TABLE OF ty_archiving_class WITH UNIQUE KEY name.
+  TYPES:
+    "! <p class="shorttext">Archiving Object</p>
+    "! Archiving Object (AOBJ) v1
+    BEGIN OF ty_main,
+      "! $required
+      format_version        TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                TYPE zif_aff_types_v1=>ty_header_60_cloud,
+      "! <p class="shorttext">Attributes of an Archiving Object</p>
```

Comment:
I guess we can also shorten this title, if we stick to the field `attributes`, can't we?

```suggestion
      "! <p class="shorttext">Attributes</p>
```

Reply:
Yes will change that.